src/Init/Meta.lean

```diff
@@ -75,7 +75,7 @@ See #2572.
 opaque Internal.hasLLVMBackend (u : Unit) : Bool
 
 /-- Valid identifier names -/
-def isGreek (c : Char) : Bool :=
+@[inline] def isGreek (c : Char) : Bool :=
```

Comment:
What's the rule for whether to inline or not inline these functions? Why is there e.g. no `inline` on `isSubScriptAlnum`?